src/scalper.ts

```diff
@@ -1030,10 +1030,10 @@ Spread % ${(whaleAskPrice - whaleBidPrice) / fairValue * 100}`);
     const whaleBidDiff = fairValue - whaleBidPrice;
     const whaleAskDiff = whaleAskPrice - fairValue;
     const backBidPrice = whaleBidDiff > fairValue - gammaBid
-      ? Math.floor((whaleBidPrice + this.tickSize) * (1 / this.tickSize)) / (1 / this.tickSize)
+      ? Math.max(Math.floor((whaleBidPrice + this.tickSize) * (1 / this.tickSize)) / (1 / this.tickSize), gammaBid * (1 - whaleMaxSpread))
```

Comment:
I will go through later and fix this logic. Really shouldnt have complicated computations like this.

Fine for now.

Reply:
Next step is to prevent trading below/above strikes depending if ITM & stuck long or short. I'll try to simplify that as much as possible. Ie. don't bid MNGO above 0.02 strike until we're net long